src/Bag.ts

```diff
@@ -23,17 +25,30 @@ export type ReadOptions = {
   freeze?: boolean;
 };
 
+export type BagOpt = {
+  decompress?: Decompress;
+  noParse?: boolean;
+};
+
+export type IteratorOpt = {
+  position?: Time;
```

Comment:
`startTime`?

Reply:
I had hoped to avoid the two word capitalization dilemma. I'll go with `start` in the public api.

Reply:
Is it a dilemma? Don't we use camelCase in JS?